README.md

```diff
@@ -190,13 +130,29 @@ openssl s_server -cert rsa.crt -key rsa.tss -keyform engine -engine tpm2tss -acc
 
 For ECDSA keys however, the Hash algorithm needs to be specified because the TPM
 does not support SHA512. You can blacklisting SHA512 universally. That is
-possible via openssl.cnf. See the "SignatureAlgorithms" configuration file command on this page:
+possible via openssl.cnf. See the "SignatureAlgorithms" configuration file
+command on this page:
 https://www.openssl.org/docs/man1.1.1/man3/SSL_CONF_cmd.html
 
-# Project layout
+Note: Usage of s_server with HSM-protected private keys is only supported on
+OpenSSL 1.1.0 and newer.
+
+## Development prefixes
+In order to use this engine without `make install` for testing call:
 ```
-├── doc     : documentation and man pages
-├── include : include files for system-wide installation
-├── src     : the source files
-└── test    : integration tests
+export LD_LIBRAY_PATH=${TPM2TSS}/src/tss2-{tcti,mu,sys,esys}/.libs:${PWD}/.libs
+export OPENSSL_ENGINES=${PWD}/.libs
+export PATH=${PWD}:${PATH}
+export PKG_CONFIG_PATH=$PWD/../tpm2-tss/lib
+./bootstrap
+./configure \
+    CFLAGS="-I$PWD/../tpm2-tss/include" \
+    LDFLAGS="-L$PWD/../tpm2-tss/src/tss2-{esys,sys,mu,tcti}/.libs"
+make
+tpm_server
+make check
 ```
+make check will use any available TPM (including /dev/tpm0,
```

Comment:
We can use the environment variable `TPM2TSSENGINE_TCTI` from #62 to force the tests to use the simulator, which is what I do [for the AUR package](https://aur.archlinux.org/cgit/aur.git/tree/tpm2-tss-engine-git_check.sh?h=tpm2-tss-engine-git). I have been meaning to submit a PR to properly integrate this into the test suite, but haven't gotten around to doing it yet. I don't think this is a blocker for the release, though.

Reply:
Love to see it. And yes, let's queue that up for 1.1.
Also feel free to open an enhancement issue about this...

Reply:
Will do!